nodejs/week1/Node-week1-Homework/meal-sharing.app/src/backend/routes/meal.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("../data/meals.json");
+const reviews = require("../data/reviews.json");
+
+
+router.get("/meal", (request, response) => {
+	meals.forEach((meal) => {
+		meal.reviews = reviews.filter((foodreview) => foodreview.mealId === meal.id);
+		return meal
+	});
+	response.json(meals[Math.floor(meals.length * Math.random())]);
```

Comment:
It is not an error!! But maybe it is a good idea to put meals[Math.floor(meals.length * Math.random())] in some variable and send it in response.json for better code readability.

Reply:
ok